xunit.runner.visualstudio.nuspec

```diff
@@ -41,5 +41,8 @@ Supported platforms:
     <file src="build\xunit.runner.visualstudio.wpa81.targets"                                   target="build\wpa81\xunit.runner.visualstudio.targets" />
     <file src="xunit.runner.visualstudio.wpa81\bin\Release\xunit.runner.visualstudio.wpa81.dll" target="build\wpa81\" />
     <file src="xunit.runner.visualstudio.wpa81\bin\Release\xunit.runner.visualstudio.wpa81.pri" target="build\wpa81\" />
+
+    <file src="xunit.runner.visualstudio.testadapter.dotnetcore\bin\Release\netstandard1.5\xunit.runner.visualstudio.testadapter.deps.json" target="build\netstandard1.5\" />
+    <file src="xunit.runner.visualstudio.testadapter.dotnetcore\bin\Release\netstandard1.5\xunit.runner.visualstudio.testadapter.dll" target="build\netstandard1.5\" />
```

Comment:
I see, I've updated the framework to netcoreapp1.0.
Since we have a separate desktop adapter, we can target dotnetcore adapter to netcoreapp1.0.
We can revisit this in future, if required.